README.md

```diff
@@ -21,34 +21,13 @@ Start a new [KMM](https://kotlinlang.org/docs/mobile/create-first-app.html) proj
 
 *See [Config.kt](buildSrc/src/main/kotlin/Config.kt#L2txt) or the [realm-kotlin releases](https://github.com/realm/realm-kotlin/releases) for the latest version number.*
 
-- Add the following Gradle configuration in the root project (make sure you're using Kotlin `1.4.20` or recent)
-`<root project>/build.gradle.kts`
-```Gradle
-buildscript {
-    repositories {
-        mavenCentral()
-    }
-    dependencies {
-        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.20")// minimum 1.4.20
-        classpath("com.android.tools.build:gradle:4.0.1")
-        classpath("io.realm.kotlin:gradle-plugin:<VERSION>")
-    }
-}
-
-allprojects {
-    repositories {
-        mavenCentral()
-    }
-}
-```
-
-- Apply the `io.realm.kotlin` plugin and specify the dependency in the common source set.
+- In the shared module (`shared/build.gradle.kts`), apply the `io.realm.kotlin` plugin and specify the dependency in the common source set.
```

Comment:
Not sure this enough? People still need to manually add `gradlePluginPortal()` and `mavenCentral()` to build script repositories and `mavenCentral()` to project repositories?

Reply:
I tested it with a fresh KMM project. `mavenCentral` and `gradlePluginPortal()` are already included in the root Gradle project.
```Gradle
buildscript {
    repositories {
        gradlePluginPortal()
        google()
        mavenCentral()
    }
    dependencies {
        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.10")
        classpath("com.android.tools.build:gradle:7.0.0-beta04")
    }
}

allprojects {
    repositories {
        google()
        mavenCentral()
    }
}

tasks.register("clean", Delete::class) {
    delete(rootProject.buildDir)
}
```

Reply:
Okay, just wasn't sure if we should post the entire setup or just assume that people are using KMM project tutorials? But not a big deal.

Reply:
I can confirm that `mavenCentral()` and `gradlePluginPortal()` are already in there using the KMM plugin template. However, I wonder if it's worth mentioning them here so that users at least know where the dependencies are coming from if they do choose to create a project without the template, or if the template changes in the future.

Reply:
I think it's fair to assume these are already in since even if the project is created from scratch it needs the `gradlePluginPortal` to apply Android and Multiplatform plugins. We will also point to the detailed doc on the website to cater for different use cases.